client/src/pages/FlightData/tabs/FlightPlan/FlightPlanToolbar.js

```diff
@@ -71,6 +91,111 @@ const FlightPlanToolbar = props => {
 		}
 	}, [props.getters.placementMode, props.getters.placementType, props.getters.firstJump, props.getters.firstPoint, props.getters.currentDistance])
 
+	const addWaypoint = (lat, lon, alt) => {
+		// Create a new waypoint
+		console.log(parseInt(alt))
+		
+		let cmds=0
+		let point;
+		if (props.getters.placementType== "waypoint"){
+			cmds=Commands.waypoint
+			point = {
+				alt:alt!="" ? parseFloat(alt):props.getters.defaultAlt,
+				cmd:cmds,
+				lat:parseFloat(lat),
+				lng:parseFloat(lon),
+				num:props.getters.path.length + 1,
+				p1:0,
+				p2:0,
+				p3:0,
+				p4:0
+	
+			};
+		}
+		else if (props.getters.placementType=="jump"){
+			return
+		}
+		else if (props.getters.placementType=="unlimLoiter"){
+			cmds=Commands.unlimLoiter
```

Comment:
is there a point in defining this then setting cmd to this? if not, I don't really see a reason for it